src/vs/workbench/contrib/chat/browser/chatWidget.ts

```diff
@@ -889,11 +889,6 @@ export class ChatWidget extends Disposable implements IChatWidget {
 			if (e.kind === 'setAgent') {
 				this._onDidChangeAgent.fire({ agent: e.agent, slashCommand: e.command });
 			}
-			if (e.kind === 'addRequest') {
-				if (this._location.location === ChatAgentLocation.EditingSession) {
-					this.chatEditingService.createSnapshot(e.request.id);
-				}
-			}
```

Comment:
Why do we not need to take snapshots anymore?

Reply:
No change, but it moved to inside the chatEditingService: https://github.com/microsoft/vscode/blob/862ef82282bd6cb2d533627b53db5c9d687ab77c/src/vs/workbench/contrib/chat/browser/chatEditing/chatEditingService.ts#L313

Reply:
Ah, thanks